### DISCUSSION TITLE
Cross-Domain Storage Solution

### DISCUSSION THREAD

**Discussion by iamnewton** [MagniteEngineering/fledge.polyfill#7] [Specification]
We know that we're going to have to store the interest groups and auction data within a  [Cross-Domain Sharing](https://github.com/aviboy2006/cross-domain-cookie-sharing) solution, but the specifics of that are not yet determined.

Should we use `localStorage`, or `sessionStorage`, or `indexedDB`, or something else?

And depending on the answer to that question, do we store in JSON?

**Comment by brodrigu** [comment]
This [article](https://web.dev/storage-for-the-web/) provides a great high-level overview of available options.

It seems like indexedDB is most obvious starting point due to its asynchronous nature, support for transactions, and much larger storage capacity.

**Reply by iamnewton** [reply]
Based on #18, we're going to go with `IndexedDB`.
